parsl/executors/high_throughput/executor.py

```diff
@@ -524,6 +524,11 @@ def submit(self, func, *args, **kwargs):
         Returns:
               Future
         """
+        if len(resource_specification) > 0:
+            logger.warning("Ignoring the resource specification. "
+                           "Parsl resource specification is not supported in HighThroughput Executor. "
+                           "Please check WorkQueueExecutor if resource specification is needed.")
```

Comment:
I would prefer that this raises an error, rather than logs a message: the user is asking the executor to do a thing that it cannot do.